src/Pods/Integrations/WPML.php

```diff
@@ -0,0 +1,335 @@
+<?php
+
+namespace Pods\Integrations;
+
+use Pods\Integration;
+
+/**
+ * Class WPML
+ *
+ * @since 2.8.0
+ */
+class WPML extends Integration {
+
+	/**
+	 * @inheritdoc
+	 */
+	protected $hooks = [
+		'action' => [],
+		'filter' => [
+			'pods_get_current_language' => [ 'pods_get_current_language', 10, 2 ],
+			'pods_api_get_table_info' => [ 'pods_api_get_table_info', 10, 7 ],
+			'pods_data_traverse_recurse_ignore_aliases' => [ 'pods_data_traverse_recurse_ignore_aliases', 10 ],
+			'pods_pods_field_get_metadata_object_id' => [ 'pods_pods_field_get_metadata_object_id', 10, 4 ],
+			'pods_component_i18n_admin_data' => [ 'pods_component_i18n_admin_data' ],
+			'pods_component_i18n_admin_ui_fields' => [ 'pods_component_i18n_admin_ui_fields', 10, 2 ],
+		],
+	];
+
+	/**
+	 * Whether the plugin is active.
+	 *
+	 * @since 2.8.0
+	 *
+	 * @return bool
```

Comment:
Lack of phpdoc is a concern here but we can revisit this in 2.8.1, let's just not forget to add phpdoc summary text for all parameters and return values.

Reply:
What do you mean here? It has phpdoc